integration-tests/gradle/src/main/resources/it-test-basic-project/src/test/java/org/acme/ExampleResourceTest.java

```diff
@@ -0,0 +1,29 @@
+package org.acme;
+
+import io.quarkus.test.junit.QuarkusTest;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.CoreMatchers.is;
+
+@QuarkusTest
+public class ExampleResourceTest {
+
+    @Test
+    public void testHelloEndpoint() {
+        given()
+          .when().get("/hello")
+          .then()
+             .statusCode(200)
+             .body(is("Hello from Test"));
```

Comment:
I don't understand how this test passes? `ExampleResource` only has a single endpoint, `/hello`, which returns `"Hello World"`?